README.md

```diff
@@ -112,3 +121,146 @@ make
+|A     | 0   | ✓           | ✓   |     |     |      |      |    |    |           |            |
+|A     | 1   | ✓           |     | ✓   |     |      |      |    |    |           |            |
+|A     | 2   | ✓           |     |     | ✓   |      |      |    |    |           |            |
+|A     | 3   | ✓           |     |     |     | ✓    |      |    |    |           |            |
+|A     | 4   | ✓           |     |     |     |      | ✓    |    |    | ✓         |            |
+|A     | 5   | ✓           |     |     |     |      |      | ✓  |    |           |            |
+|A     | 6   | ✓           |     |     |     |      |      |    | ✓  |           |            |
+|A     | 7   | ✓           |     |     |     |      |      |    |    | ✓         |            |
+|B     | 0   | ✓           | ✓   |     |     |      |      |    |    | ✓         |            |
+|B     | 1   | ✓           |     | ✓   |     |      |      |    |    | ✓         |            |
+|B     | 2   | ✓           |     |     | ✓   |      |      |    |    | ✓         |            |
+|B     | 3   | ✓           |     |     |     | ✓    |      |    |    | ✓         |            |
+|B     | 4   | ✓           |     |     |     |      | ✓    |    |    | ✓         |            |
+|B     | 5   | ✓           |     |     |     |      |      | ✓  |    | ✓         |            |
+|B     | 6   | ✓           |     |     |     |      |      |    | ✓  | ✓         |            |
+|B     | 7   | ✓           |     |     |     |      |      |    |    | ✓         | ✓          |
+
+All pins (0-7) can be used as digital pins. All pins on port B and pin 7 on port A can be used as analog pins.
+
+If you're newer to hardware and these functions look like alphabet soup to you, take a look at our [communication protocols documentation](https://tessel.io/docs/communicationProtocols) to get an idea of how these pins should be used.
+
+### Digital pins
+
+A digital pin (any pin other than 3.3V and GND on Tessel 2) is either high (on/3.3V) or low (off/0V). On both of ports A and B, pins 0 and 1 are pulled high to 3.3V by default.
+
+Here is an example usage of a digital pin on Tessel:
+
+```js
+var tessel = require('tessel'); // import tessel
+var myPin = tessel.port.A.pin[2]; // select pin 2 on port A
+myPin.output(1);  // turn pin high (on)
+console.log(myPin.read()); // print the pin value to the console
+myPin.output(0);  // turn pin low (off)
+```
+
+### Analog pins
+
+An analog pin is a pin whose value can vary in the range between 0V and 3.3V. Pins 4 and 7 on port A and all pins on port B can read analog values (though pins 0 and 1 are pulled to 3.3V by default and are thus not recommended for this purpose). Pin 7 on port B can write an analog value.
+
+Here is an example usage of an analog pin on Tessel:
+
+```js
+var tessel = require('tessel'); // import tessel
+var myPin = tessel.port.B.pin[7]; // select pin 2 on port A
+myPin.analogWrite(.6);  // turn pin to 60% of high
```

Comment:
Since there are already a few minor nits to fix, I suggest changing this to `0.6`